packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+// 3rd party
+import {
+  Link as ReactAriaLink,
+  LinkProps as ReactAriaLinkProps,
+} from "react-aria-components";
+
+import { WalkthroughInfo } from "@repo/data/useWalkthroughData";
+import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps extends ReactAriaLinkProps {
+  id: string;
+  data: WalkthroughInfo;
+  walkthroughId: string;
+}
+
+export default function WalkthroughCard({
+  id,
+  data,
+  walkthroughId,
```

Comment:
Since we're extending the props, we should probably catch them here with `...props` and pass them along to ReactAriaLink.